src/application-status-overlay/ApplicationStatusOverlay.jsx

```diff
@@ -0,0 +1,69 @@
+import { useRef, useContext, useLayoutEffect } from 'react';
+import PropTypes from 'prop-types';
+import Button from 'terra-button';
+import uuidv4 from 'uuid/v4';
+import ApplicationStatusOverlayContext from './ApplicationStatusOverlayContext';
+
+const propTypes = {
+  /**
+   * An array of objects containing terra-button properties. A key attribute is required for each object.
+   * This array is used to render buttons in the bottom section.
+   * Example:`[{ text: 'Button 1', key: 1, size: 'medium', variant: 'action', onClick: onClickFunction}]`
```

Comment:
Example needs to be updated - there is no longer `size` as a button prop and `action` button should not be recommended or suggested to be used here, only neutral or emphasis, most likely to be removed later anyway.
```suggestion
   * Example:`[{ text: 'Refresh Application', key: 1, variant: 'neutral', onClick: myRefreshFunction}]`
```

Reply:
Changed here : https://github.com/cerner/terra-application/pull/47/commits/62b43a41763ab55e834f07ce82e374ac33e0b409
Also, updated the **terra-status-view** documentation and references as well : https://github.com/cerner/terra-core/pull/3099